.github/workflows/github-ci.yml

```diff
@@ -26,5 +26,14 @@ jobs:
                   ./mvnw -ntp clean verify -P-webpack
             - name: Run frontend test
               run: npm run test
+            - name: Sonar analysis
```

Comment:
@avdev4j : I configured SonarCloud, that's why I'm merging this. Anyway, you need to exclude the Sonar analysis when it's not master branch, or when it's in a fork. Otherwise, it will fail as the user won't have SONAR_TOKEN